datatypes/src/raster/operations/classification.rs

```diff
@@ -0,0 +1,84 @@
+use crate::raster::{data_type::FromPrimitive, BaseRaster, GridDimension, Pixel, Raster};
+use crate::util::Result;
+use num_traits::AsPrimitive;
+
+pub trait RasterClassification<D, C>
```

Comment:
Why introduce a trait instead of adding method to BaseRaster? For what other structs would we implement this trait?

Reply:
RasterTile, ...